openprescribing/measure_definitions/carbon_salbutamol.json

```diff
@@ -0,0 +1,81 @@
+    "committed to reducing its carbon footprint by 51% by 2025</a> to meet the target in the Climate Change Act, including a shift to lower carbon inhalers",
+    "to deliver a reduction of 4%.</p>",
+    "<p>Salbutamol metered dose inhalers (MDIs) are the single biggest source of carbon emissions from NHS medicines prescribing.",
+    "The 2022-2023 PCN Investment and Impact Fund (IIF) therefore ",
+    "<a href='https://www.england.nhs.uk/publication/network-contract-directed-enhanced-service-investment-and-impact-fund-2022-23-updated-guidance/'>",
+    "includes an indicator</a> focusing on mean carbon emissions per salbutamol inhaler prescribed (kg CO<sub>2</sub>e).",
+    "The aim is to encourage a change in prescribing of salbutamol devices to those associated with lower carbon emissions, resulting in a significant reduction in NHS carbon emissions.</p>",
+    "<p>Data on carbon emissions per salbutamol inhaler can be viewed in the ",
+    "<a href='https://docs.google.com/spreadsheets/d/1eyk2iqNM_4sobDzevZWWcRR00O2tmJO5HwdFZLjhfa4'>",
+    "table here</a>.</p>"
+  ],
+  "tags": [
+    "core",
+    "respiratory"
+  ],
+  "url": null,
+  "is_percentage": false,
+  "is_cost_based": false,
+  "low_is_good": true,
+  "numerator_type": "custom",
+  "numerator_columns": [
+    "SUM(",
+    "CASE WHEN bnf_code = '0301011R0AAAPAP' THEN quantity * 25.24 --Salbutamol 100micrograms/dose inhaler CFC free \n",
+    "WHEN bnf_code = '0301011R0AABUBU' THEN quantity * 11.79 --Salbutamol CFC free breath actuated inhaler 100 micrograms \n",
+    "WHEN bnf_code = '0301011R0AAAQAQ' THEN quantity * 0.58 --Salbutamol 200micrograms/dose dry powder inhaler derived from Ventolin Accuhaler \n",
+    "WHEN bnf_code = '0301011R0BEAHAQ' THEN quantity * 0.58 --Ventolin 200micrograms/dose Accuhaler \n",
+    "WHEN bnf_code = '0301011R0BEAIAP' THEN quantity * 28.26 --Ventolin 100micrograms/dose Evohaler \n", 
+    "WHEN bnf_code = '0301011R0BIAFAP' THEN quantity * 11.95 --Salamol 100micrograms/dose inhaler CFC free (Teva) \n",
+    "WHEN bnf_code = '0301011R0BIAHAP' THEN quantity * 11.95 --Salamol 100micrograms/dose inhaler CFC free (Arrow) \n",
+    "WHEN bnf_code = '0301011R0BIAGBU' THEN quantity * 12.08 --Salamol 100micrograms/dose Easi-Breathe inhaler \n",
+    "WHEN bnf_code = '0301011R0BMAAAP' THEN quantity * 9.72 --Airomir 100micrograms/dose inhaler \n",
+    "WHEN bnf_code = '0301011R0BMABBU' THEN quantity * 9.72 --Airomir Autohaler \n",
+    "WHEN bnf_code = '0301011R0BWAABZ' THEN quantity * 0.62 --Easyhaler Salbutamol sulfate 100micrograms/dose dry pdr inh \n",
+    "WHEN bnf_code = '0301011R0BWABAQ' THEN quantity * 0.62 --Easyhaler Salbutamol sulfate 200micrograms/dose dry pdr inh \n",
+    "WHEN bnf_code = '0301011R0BXAACB' THEN quantity * 3.75 --Salbulin Novolizer 100micrograms/dose inhalation powder \n",
+    "WHEN bnf_code = '0301011R0BXAACB' THEN quantity * 3.75 --Salbulin Novolizer 100micrograms/dose inhalation powder refill \n",
```

Comment:
BNF code on line 47 and 48 repeat themselves.

I think this should be `0301011R0BXABCC`